### PR TITLE
SignUp

### PR DESCRIPTION
**Pull request by 160hanzen**
I believe this is done

### CODE REVIEW

File: screens/user/SignUp.js [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+                    />
+                    <NHSInput 
+                        mode='outlined'
+                        label='E-mail' 
+                    />
+                    <NHSInput 
+                        mode='outlined'
+                        label='Password' 
+                        value={this.StaticRange.password}
+                        onChangeText={ (Password) => this.setState({Password})}
```
Comment (AliKayahan) [inline screens/user/SignUp.js]: Please define state manager function before component return() ex.

const [password, setPassword] = useState('');

File: screens/user/SignUp.js [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+        justifyContent: 'center'
+    },  
+    signupTextContainer: {
+        marginTop: 10,
+        width: '85%',
+        alignContent: 'center',
+        justifyContent: 'center',
+        alignItems: 'center'
+    },
+    signUpText: {
```
[the remark this answers is not in the document]
Comment (160hanzen) [reply screens/user/SignUp.js]: Which file do I use instead of the const stylesheet?

[the remark this answers is not in the document]
Comment (160hanzen) [reply screens/user/SignUp.js]: Ah so, dont put anything in their unless it's for the default stylsheet container?

[the remark this answers is not in the document]
Comment (AliKayahan) [reply screens/user/SignUp.js]: We should group app-wide stylings into NHSStyle.js to keep a compact look and feel at each screen we are working separately. So if a text type doesn't exist you can create but for this instance, we already have a header styling that you can use which I also used for login screen.

File: screens/user/SignUp.js [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+const SignUpScreen = () => {
+    return(  //calling the background from the directory
+        <Background>
+            <View style={styles.logoContainer}>
+                <Image 
+                    source={require('../assets/images/NHSLogo.png')} 
+                    />
+            </View>
+            <View style={styles.bodyContainer}>
+                <Text style={NHSStyle.header1}>
```
[the remark this answers is not in the document]
Comment (160hanzen) [reply screens/user/SignUp.js]: Do I just do a pull request from terminal to get the proper nhsstylesheet?

[the remark this answers is not in the document]
Comment (AliKayahan) [reply screens/user/SignUp.js]: Yes, you can switch to your master branch, run git pull then switch back to your SignUp branch and run git merge master to update your current branch.

[the remark this answers is not in the document]
Comment (160hanzen) [reply screens/user/SignUp.js]: I see, thanks

**Pull request closed without merging**